godot-core/src/engine/gfile.rs

```diff
@@ -739,11 +788,11 @@ impl BufRead for GFile {
         let gd_buffer = self.fa.get_buffer(buffer_read_size as i64);
         self.check_error()?;
 
-        for i in 0..gd_buffer.len() {
-            self.buffer[i] = gd_buffer.get(i);
-        }
+        let readen_buffer = &mut self.buffer[0..gd_buffer.len()];
+
+        readen_buffer.copy_from_slice(gd_buffer.as_slice());
 
-        Ok(&self.buffer[0..gd_buffer.len()])
+        Ok(readen_buffer)
```

Comment:
What does "readen" mean, it's not a word...?

I'd just use `read_buffer`.

Reply:
Will fix it. It seems I came from past 😆 

https://en.m.wiktionary.org/wiki/readen